src/web/components/SeriesSectionLink.tsx

```diff
@@ -126,6 +128,12 @@ export const SeriesSectionLink = ({
 											color: ${palette.text.seriesTitle};
 											background-color: ${palette
 												.background.seriesTitle};
+											box-shadow: -6px 0 0 0
+													${palette.background
+														.seriesTitle},
+												6px 0 0 0
+													${palette.background
+														.seriesTitle};
```

Comment:
I'm applying this everywhere so that we sematically stretch all backgrounds 6px either side but when transparent this has no noticable effect.

I'm also keeping it verbose and inline because I expect this component to be refactored soon and want to make that easier